controllers/v2/unavailability/api.py

```diff
@@ -64,7 +69,26 @@ def get(self, user_id):
     def post(self, user_id):
         try:
             args = edit_parser.parse_args()
+            # Check if start time is earlier than end time
+            if args['start'] >= args['end']:
+                return {"description": "Start time must be earlier than end time"}, 400  # HTTP 400 Bad Request
```

Comment:
"message" rather than "description" is more in line with the rest of the app (please do this to the bit below as well)